gui/slick/interfaces/default/IRC.mako

```diff
@@ -1,15 +1,17 @@
-#import sickbeard
-#from sickbeard.common import *
+<%
+    import sickbeard
+    from sickbeard.common import *
 
-#import os.path
-#include $os.path.join($sickbeard.PROG_DIR, "gui/slick/interfaces/default/inc_top.tmpl")
+    import os.path
+    include $os.path.join($sickbeard.PROG_DIR, "gui/slick/interfaces/default/inc_top.tmpl")
```

Comment:
Nope, mako only needs it for inline HTML. Check the next commit. https://github.com/SiCKRAGETV/SickRage/commit/4348f6f8d429237f79ec97a5428cc7a28795f929